### PR TITLE
XW-3300 | DS community header fixes

### PR DESCRIPTION
**Pull request by weronika**
https://wikia-inc.atlassian.net/browse/XW-3470
https://wikia-inc.atlassian.net/browse/XW-3473
https://wikia-inc.atlassian.net/browse/XW-3474
https://wikia-inc.atlassian.net/browse/XW-3475

### CODE REVIEW

File: dist/scss/wds-components/_tabs.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
 .wds-tabs__tab {
 	$tab-padding-horizontal: 12px;
 	$tab-padding-vertical: 13px;
```
Comment (latata) [inline dist/scss/wds-components/_tabs.scss]: Looks like that variable is not used anymore

File: dist/scss/wds-components/_tabs.scss [hunk 2]
```diff
@@ -45,7 +48,7 @@
 		transition: border-color .1s;
 
 		.wds-tabs__tab-label {
-			padding-bottom: $tab-padding-vertical - 2px;
+			height: $tab-height - 4px;
```
Comment (latata) [inline dist/scss/wds-components/_tabs.scss]: what's `4px` for? maybe we should add comment here?

File: dist/scss/wds-components/_tabs.scss [hunk 3]
```diff
@@ -45,7 +47,9 @@
 		transition: border-color .1s;
 
 		.wds-tabs__tab-label {
-			padding-bottom: $tab-padding-vertical - 2px;
+			// 4px is a height needed to align text after adding 2px bottom-border - it is 2 * 2px
+			// (we need to subtract 2px from bottom and top of the label container for text to be aligned properly)
+			height: $tab-height - 4px;
```
Comment (hakubo) [inline dist/scss/wds-components/_tabs.scss]: can we always show border with transparent color? then we would not have to calculate the size here

Comment (weronika) [inline dist/scss/wds-components/_tabs.scss]: good idea 👍

File: style-guide/styles/wds-components/_tabs.scss [hunk 4]
```diff
@@ -53,7 +53,7 @@
 	}
 
 	.wds-dropdown__content {
-		left: -$wds-dropdown-padding;
+		left: -$wds-dropdown-padding + $tab-margin-horizontal;
```
Comment (hakubo) [inline style-guide/styles/wds-components/_tabs.scss]: tab-margin-horizontal to left? are we sure about it?

**Pull request merged**